### PR TITLE
New package: sigi-3.1.0

### PR DESCRIPTION
**Pull request by hiljusti**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

Note: I am the developer of [sigi](https://github.com/hiljusti/sigi), and rely on it daily for my personal organization.

### CODE REVIEW

**Comment by ericonr** [comment]
The commit message is still outdated, you can use `git commit --ammend` to fix it. Please avoid opening new PRs for the same package, as well: just force push to the PR with the new version.

**Comment by hiljusti** [comment]
I udated the commit message and also rebased once. (I'll rebase again if needed.)

**Comment by cinerea0** [comment]
Small nitpick, but I'd suggest changing your short description from "Sigi is an organizing tool and no-frills stack database" to "Organizing tool and no-frills stack database" to fit with the style of other package short descriptions. Also, you don't need to check off anything under the "Does it build and run successfully?" section or even display it because your PR can be processed through CI in a reasonable amount of time.

**Comment by hiljusti** [comment]
Thanks @cinerea0 ! I updated

**Comment by ericonr** [comment]
The template looks ok, but for new packages I'd like to wait a bit, see how both maintenance and interest go.

**Comment by hiljusti** [comment]
Thanks @ericonr!

For what it's worth, I'm the developer, and my daily driver is void. I use it to help me organize my life and don't plan to switch to other software.

**Comment by hiljusti** [comment]
@ericonr As far as maintenance/interest, I think both are going pretty well here?

- I've gotten past two major version bumps (including backwards compat migration tools)
- Downloads of Sigi are up to 800 (from 350 back in April). I'm not advertising it, so this is just Rust folks who are finding it organically in crates.io and giving it a shot (or updating)
- I've also started getting (and responding to) issues on the project.

I know "TO DO list" is a pretty basic idea, and there's [plenty of other implementations](https://github.com/hiljusti/sigi/blob/core/NOTES.md). That said, Sigi strikes a really good balance between being simple, scriptable, and useful when compared to other tools.

**Comment by hiljusti** [comment]
Fixed lint failure (The description was longer than 72 chars)

**Comment by hiljusti** [comment]
@ericonr Could I ask you to take a look again? (Only because you've looked before)

**Comment by ericonr** [comment]
Can you please rebase for CI to run correctly?

**Comment by hiljusti** [comment]
@ericonr Rebased

**Comment by hiljusti** [comment]
(I'll just keep rebasing periodically)

**Comment by hiljusti** [comment]
@ericonr How does it look?

**Comment by hiljusti** [comment]
Note: License changed in version 3.0.0.

It used to be GPL3+, now it is GPL2 (only)

**Review by subnut — CHANGES_REQUESTED** [review]

File: srcpkgs/sigi/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'sigi'
+pkgname=sigi
+version=3.1.0
+revision=1
+build_style=cargo
+short_desc="Organization tool for people who hate organization"
+maintainer="J.R. Hill <hiljusti@so.dang.cool>"
+license="GPL-2.0-only"
+homepage="https://crates.io/crates/sigi"
+distfiles="https://crates.io/api/v1/crates/sigi/${version}/download>sigi-${version}.tar.gz"
```
[the remark this answers is not in the document]
Comment (hiljusti) [reply srcpkgs/sigi/template:10]: I can do this - is there a reason to prefer github over crates.io though?

Github tags can change, but crate identifiers are immutable. I kind of prefer the former

File: srcpkgs/sigi/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+checksum=967e492a89f05c3a373929a48e0a1a7712d4742b150216228feff4a7869297e3
+
+do_check() {
+	# CLI tests in the project rely on BATS (more than just bats-core) through
+	# git submodules. We skip those tests here until the other bats components
+	# are imported in Void or https://github.com/hiljusti/sigi/issues/19
+	SKIP_BATS_TESTS=1 cargo test --release --target ${RUST_TARGET}
+}
```
[the remark this answers is not in the document]
Comment (hiljusti) [reply srcpkgs/sigi/template:18]: Ohh great call. Thanks!

File: srcpkgs/sigi/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiljusti) [reply srcpkgs/sigi/template:10]: (The "crate" from Rust is also a source tarball)

File: srcpkgs/sigi/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (hiljusti) [reply srcpkgs/sigi/template]: Made this change

File: srcpkgs/sigi/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiljusti) [reply srcpkgs/sigi/template]: I'll update the homepage, but right now I want to keep the distfile at crates.io. The link is immutable, and publishing to crates.io has some really basic validation, like the sources being compilable

**Comment by hiljusti** [comment]
@subnut How's it look?

**Pull request merged**